sklearn/linear_model/_coordinate_descent.py

```diff
@@ -612,7 +612,7 @@ def enet_path(
             precompute,
             fit_intercept=False,
             copy=False,
-            check_input=check_input,
+            check_gram=True,
```

Comment:
why not `check_gram=check_input` ?

the description of `check_input` for `enet_path` is
```If set to False, the input validation checks are skipped (including the
        Gram matrix when provided). It is assumed that they are handled
        by the caller.
```

Reply:
ah right, missed that :)